assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -78,7 +88,12 @@ const CardTitle = ({
   onEditComplete,
   onEditCancel,
   tabIndex,
+  locked = false,
+  lockUser = 0,
```

Comment:
Would it be possible to have more than just the user ID here?

In case we want to show an avatar and something like "John Doe is currently editing"

Reply:
I didn't add this, as there is no UX for it and didn't really feel like part of an MVP. MVP is meant to be to test an idea works, not be fully featured.

Reply:
But would it be _possible_, i.e. do we have the data?

Reply:
We didn't have the data, but we do now. I have added it here. https://github.com/google/web-stories-wp/pull/6497/commits/fc05a16dceebe53d66fd61f4fe725961fe69ec5c

<img width="282" alt="Screenshot 2021-02-25 at 11 59 16" src="https://user-images.githubusercontent.com/237508/109150803-451d1900-7761-11eb-9147-4a83d2e87e23.png">